src/MunicipalityRegistry.Api.CrabImport/Infrastructure/Startup.cs

```diff
@@ -43,23 +43,33 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
         {
             services
                 .ConfigureDefaultForApi<Startup>(
-                    (provider, description) => new Info
+                    new StartupConfigureOptions
                     {
-                        Version = description.ApiVersion.ToString(),
-                        Title = "Basisregisters Vlaanderen Municipality Registry API",
-                        Description = GetApiLeadingText(description),
-                        Contact = new Contact
+                        Cors =
                         {
-                            Name = "Informatie Vlaanderen",
-                            Email = "informatie.vlaanderen@vlaanderen.be",
-                            Url = "https://legacy.basisregisters.vlaanderen"
+                            Origins = _configuration
+                                .GetSection("Cors")
+                                .GetChildren()
+                                .Select(c => c.Value)
+                                .ToArray()
+                        },
+                        Swagger =
+                        {
+                            ApiInfo = (provider, description) => new Info
+                            {
+                                Version = description.ApiVersion.ToString(),
+                                Title = "Basisregisters Vlaanderen Municipality Registry API",
+                                Description = GetApiLeadingText(description),
+                                Contact = new Contact
+                                {
+                                    Name = "Informatie Vlaanderen",
+                                    Email = "informatie.vlaanderen@vlaanderen.be",
+                                    Url = "https://legacy.basisregisters.vlaanderen"
+                                }
+                            },
+                            XmlCommentPaths = new [] { typeof(Startup).GetTypeInfo().Assembly.GetName().Name }
                         }
```

Comment:
Can you add Healtchecks too please:
https://github.com/Informatievlaanderen/templates/blob/master/src/Be.Vlaanderen.Basisregisters.Templates/Be.Vlaanderen.Basisregisters.Registry/src/ExampleRegistry.Api/Infrastructure/Startup.cs#L90-L105